code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -12,6 +12,130 @@
 	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
 	strip_mod = 0.9
 
+/obj/item/clothing/gloves/fingerless/pugilist
+	name = "armwraps"
+	desc = "A series of armwraps. Makes you pretty keen to start punching people."
+	icon_state = "armwraps"
+	item_state = "armwraps"
+	body_parts_covered = ARMS
+	cold_protection = ARMS
+	strip_delay = 300 //you can't just yank them off
+	var/enhancement = 0 //it's a +0 to your punches because it isn't magical
+	var/inherited_trait = TRAIT_NOGUNS //what are you, dishonoroable?
+	var/secondary_trait = TRAIT_FEARLESS //what are you, a coward?
```

Comment:
```suggestion
	///Extra damage through the punch.
	var/enhancement = 0 //it's a +0 to your punches because it isn't magical
	///Main trait added by the gloves to the user on wear.
	var/inherited_trait = TRAIT_NOGUNS //what are you, dishonoroable?
	///Secondary trait added by the gloves to the user on wear.
	var/secondary_trait = TRAIT_FEARLESS //what are you, a coward?
```
Documentation of new vars.